apps/src/StudioApp.js

```diff
@@ -697,6 +697,42 @@ StudioApp.prototype.handleSharing_ = function (options) {
   }
 };
 
+export function makeFooterMenuItems(i18n, project) {
```

Comment:
I think you can just use the i18n and project modules that gets imported rather than passing them into this function. My assumption is that you've done this to make testing easier, but using `sinon`, you can just stub the exact i18n/project module that are being imported in this file without having to pass them in.